cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -197,18 +202,39 @@ class ARROW_DS_EXPORT RadosParquetFileFormat : public ParquetFileFormat {
   std::shared_ptr<DirectObjectAccess> doa_;
 };
 
+/// \brief Serialize scan request to a bufferlist.
+/// \param[in] options Scan options.
+/// \param[in] file_size File size.
+/// \param[out] bl Output bufferlist.
+/// \return Status.
 ARROW_DS_EXPORT Status SerializeScanRequest(std::shared_ptr<ScanOptions>& options,
                                             int64_t& file_size, ceph::bufferlist& bl);
 
+/// \brief Deserialize scan request from bufferlist.
+/// \param[out] filter The filter expression to apply.
+/// \param[out] partition Partition expression.
+/// \param[out] projected_schema The projection schema.
```

Comment:
The schema to project the filtered record batches

Reply:
Done.